docs/advanced/csidriver.md

```diff
@@ -51,25 +57,105 @@ With the kubeconfig of the Harvester cluster, you can install the third-party CS
+
+![created-image-with-nfs-csi](/img/v1.5/advanced/created-image-with-nfs-csi.png) 
+
+### Virtual Machine Creation
+
+Your virtual machines can use root and data volumes in external storage.
+
+When [creating a virtual machine](../vm/create-vm.md) using the Harvester UI (**Virtual Machine > Create**), you must perform the following actions on the **Volumes** tab:
+
+- Select a virtual machine image stored in the external storage solution, and then configure the required settings.
+- Add a data volume.
+
+![various-volumes-for-vm-creating](/img/v1.5/advanced/various-volumes-for-vm-creating.png)
+
+In the following example, the root volume is created using NFS, and the data volume is created using the Longhorn V2 Data Engine.
+
+![various-volumes-for-vm-created](/img/v1.5/advanced/various-volumes-for-vm-created.png)
+
+### Volume Creation
+
+You can create volumes in your external storage solution.
+
+When [creating a volume](../volume/create-volume.md) using the Harvester UI (**Volumes > Create**), you must perform the following actions:
+
+- **Storage Class**: Select the target StorageClass, e.g. **nfs-csi**.
+- **Volume Mode**: Select the corresponding volume mode, e.g. **Filesystem** for **nfs-csi**.
+
+![create-fs-volume](/img/v1.5/advanced/create-fs-volume.png)
+
+## Advanced Topics
+
+### Storage Profiles
+
+You can now use the CDI API to create custom [storage profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) that simplify definition of data volumes. Storage profiles allow multiple data volumes to share the same provisioner settings.
+
+The following is an example of an LVM storage profile:
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: lvm-node-1-striped
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+status:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Block
+  cloneStrategy: snapshot
+  dataImportCronSourceFormat: pvc
+  provisioner: lvm.driver.harvesterhci.io
+  snapshotClass: lvm-snapshot
+  storageClass: lvm-node-1-striped
+```
+
+For more information, see [Storage Profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) in the CDI documentation.
 
-## Use the CSI driver
+You can define the above fields to override the default configuration showing on the status.
 
-After successfully configuring these settings, you can utilize the third-party StorageClass. You can apply the third-party StorageClass when creating an empty volume or adding a new block volume to a VM, enhancing your Harvester cluster's storage capabilities.
+### Limitations
 
-With these configurations in place, your Harvester cluster is ready to make the most of the third-party storage integration.
+- Backup support is currently limited to Longhorn V1 Data Engine volumes. Harvester is unable to create backups of volumes in external storage. 
```

Comment:
I think we should link this to the "VM Backup Compatibility" added in https://github.com/harvester/docs/pull/731, to provide user with more context. E.g., _"For more information, see [VM Backup Compatibility](#vm-backup-compatibility)."_